internal/services/recoveryservices/site_recovery_replicated_vm_resource.go

```diff
@@ -175,6 +176,60 @@ func resourceSiteRecoveryReplicatedVM() *pluginsdk.Resource {
 							ValidateFunc:     azure.ValidateResourceID,
 							DiffSuppressFunc: suppress.CaseDifference,
 						},
+
+						"target_disk_encryption_info": {
```

Comment:
info is redundant and doesn't add anything to ux here, so we should remove it
```suggestion
						"target_disk_encryption": {
```

Reply:
_info removed